content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/accordion.html

```diff
@@ -0,0 +1,44 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe Systems Incorporated
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<div data-sly-use.accordion="com.adobe.cq.wcm.core.components.models.Accordion"
+     data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
+     class="cmp-accordion"
+     data-cmp-is="accordion">
+    <sly data-sly-test="${accordion.items && accordion.items.size > 0}"
+         data-sly-repeat.item="${accordion.items}">
+        <div class="cmp-accordion__item${item.name == accordion.expandedItem ? ' initially-expanded' : ''}"
```

Comment:
Right now it's an element with role button wrapping the heading, but by standard this should be reversed - heading should be wrapping the button

source: https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html
quote: "Each accordion header element contains a button that controls the visibility of its content panel"

> 

>

Reply:
@brandonmaynard before going ahead with this change, I've made a second version on top of your changes which has the markup structure as @jakublamprecht describes.

Among the changes I switched `getExpandedItem` to `getExpandedItems` so that multiple items can be initially expanded and added an `isSingleExpansion` option so that the accordion can be configured to only allow one item to be expanded at a time. Will create a follow up PR probably tomorrow for us to review.

Reply:
Hey @richardhand, I've found another little issue with the markup related to the aria-expanded attribute - I've added a comment on #600 if you want to take a look at it too

Reply:
Also, do we need the initially-expanded class? Wouldn't it be possible to achieve the same result with just .cmp-accordion__item--expanded and .cmp-accordion__itempanel--expanded classes?

Reply:
@jakublamprecht - I addressed both of your above comments in #601, I replaced the initially-expanded concept with a data attribute configuration on the item - `data-cmp-expanded` and we now provide `cmp-accordion__panel--expanded` (on the panel) and `cmp-accordion__button--expanded` (on the control) classes for styling.